ble/services/DFUService.h

```diff
@@ -101,7 +103,15 @@ class DFUService {
                 handoverCallback();
             }
 
-            bootloader_start();
+            // Call bootloader_start implicitly trough a event handler call
+            // it is a work around for bootloader_start not being public in sdk 8.1
+            ble_dfu_t p_dfu;
```

Comment:
the first parameter isn't actually used in the path which calls bootloader_start(). I'll be fine if you still feel we should initialize a local variable for this. But I would also like to eliminate anything that's un-necessary.